be/extmap.c

```diff
@@ -675,22 +689,30 @@ M0_INTERNAL void m0_be_emap_obj_delete(struct m0_be_emap *map,
 				       struct m0_be_op   *op,
 				 const struct m0_uint128 *prefix)
 {
-	struct m0_be_emap_cursor it;
-	int                      rc;
+	int                       rc = -ENOMEM;
+	struct m0_be_emap_cursor *it;
+
+	M0_ALLOC_PTR(it);
```

Comment:
Can you switch the order of this line and next line?
So we can check the result of allocation immediately after allocation.

Reply:
Yeap, done. Thanks.